src/panels/lovelace/strategies/areas/areas-view-strategy.ts

```diff
@@ -0,0 +1,68 @@
+import { ReactiveElement } from "lit";
+import { customElement } from "lit/decorators";
+import type { LovelaceSectionConfig } from "../../../../data/lovelace/config/section";
+import type { LovelaceViewConfig } from "../../../../data/lovelace/config/view";
+import type { HomeAssistant } from "../../../../types";
+
+export interface AreasViewStrategyConfig {
+  type: "areas";
+}
+
+@customElement("areas-view-strategy")
+export class AreasViewStrategy extends ReactiveElement {
+  static async generate(
+    _config: AreasViewStrategyConfig,
+    hass: HomeAssistant
+  ): Promise<LovelaceViewConfig> {
+    const areas = Object.values(hass.areas).sort((a, b) => {
+      const floorA = a.floor_id ? hass.floors[a.floor_id] : null;
+      const floorB = b.floor_id ? hass.floors[b.floor_id] : null;
+      return -1 * ((floorB?.level ?? Infinity) - (floorA?.level ?? Infinity));
```

Comment:
Should they be sorted by name after floor?

Reply:
I just discussed with @marcinbauer85 we will not group by floor as user will want to reorder the areas the way they want.

Reply:
I will remove this part.